### ISSUE TITLE
Better indicate when compilation or first execution of a Flutter app is taking place

### ISSUE THREAD

**Issue by RedBrogdon** [dart-lang/dart-pad#1033]
Prior to the first rendered frame of flutter_web output and in between clicking the execute button and the first frame of that execution, the web output pane is either out of date or blank for a long pause. It would be nice to have some kind of indicator to inform the user that output will be arriving shortly.

This depends on #1032 for knowing when to remove the indicator.

**Comment by dumazy** [comment]
Maybe not the right place to ask this, but is there any possibility now for using flutter_web in DartPad?
Or is there any GitHub issue to follow this?
Thanks

**Comment by RedBrogdon** [comment]
It's not exactly the right place, but it's still a good question. 😄 

We have been working on incorporating Flutter for web into DartPad for a while now, and have released a codelab that features its use:

https://flutter.dev/docs/codelabs/layout-basics

You can complete code exercises, run them, and see Flutter for web running your code.

We haven't begun the work of integrating Flutter for web into the main DartPad UI yet, but it's on our roadmap. If you'd like to become a contributor, check out the guidelines:

https://github.com/dart-lang/dart-pad/blob/master/CONTRIBUTING.md

**Comment by RedBrogdon** [comment]
I've revised the title of this issue to better explain the current state of affairs. Right now we have indicators for when a compile is taking place (MDCLinearProgress in the embeds and a grayed-out Run button on both it and the playground). We don't have an indicator for the time between receiving compiled code and the first frame of a Flutter app appearing.

Hot restart has cut down on this problem by limiting the delay between those two things, but it can still be an annoyance for some users.

**Comment by johnpryan** [comment]
This is fixed in the new UI.
